src/components/NetworkGraph.tsx

```diff
@@ -206,7 +196,11 @@ export class NetworkGraph extends React.Component<IAllProps, {}> {
     }
 
     function updateLink(link: d3.Selection<SVGElement, any, any, any>) {
+      const k = 6 * graphLayout.alpha()
       link
+        .each(function(d) {
+          d.source.x -= k * 7, d.target.x += k * 4
+        })
```

Comment:
I'm a little confused at the use of these various constants (6, 7, and 4) -- will chat about it in person!